stac_geoparquet/arrow/_to_arrow.py

```diff
@@ -1,6 +1,5 @@
 """Convert STAC data into Arrow tables"""
 
-import ciso8601
```

Comment:
since pandas is a dependency, maybe this could be dropped to just use pandas or pyarrow for date handling? see also https://github.com/stac-utils/stac-geoparquet/pull/31#discussion_r1544730642

Reply:
Definitely don't want to use pandas here, but using pyarrow natively for the cast is fine.

Reply:
I think it's fine for ciso8601 to still be used in the tests.